src/common/path_util.cpp

```diff
@@ -82,11 +82,16 @@ static std::filesystem::path GetBundleParentDirectory() {
 
 static auto UserPaths = [] {
 #ifdef __APPLE__
-    std::filesystem::current_path(GetBundleParentDirectory());
```

Comment:
Rather than getting rid of the portable mode this should detect if the portable directory exists and use it.

Reply:
> Rather than getting rid of the portable mode this should detect if the portable directory exists and use it.

I was thinking of expanding a GUI like when user is being asked about games directory, to ask about that path as well and fill defaults with `Application Support` path but did that in a rush 😅 

Will improve it.

Reply:
I think for now it would be sufficient to just check if the portable path exists and set it to the `Application Support` path if not.

Reply:
> I think for now it would be sufficient to just check if the portable path exists and set it to the `Application Support` path if not.

Should work as proposed now.
If someone wants to move to standard Apple path, just delete or move it out of folder where emulator is.

Reply:
Should work now, even a few lines of code are not suitable on weekends for me it seems.
```sh
cd GitHub/shadps4/build                                                                         
rm -rf user
# No user folder in app location, goes for requested path:                                                                                     
./shadps4.app/Contents/MacOS/shadps4                                                               
Saving new configuration file /Users/marcinmitura/Library/Application Support/shadPS4/user/config.toml
2024-08-31 14:01:02.144 shadps4[11679:3467331] +[IMKClient subclass]: chose IMKClient_Modern
2024-08-31 14:01:02.144 shadps4[11679:3467331] +[IMKInputSession subclass]: chose IMKInputSession_Modern
rm -rf /Users/marcinmitura/Library/Application Support/shadPS4/user
clear                                                                                           
mkdir user
# Present user folder in app location, goes for old path:                                                                                         
./shadps4.app/Contents/MacOS/shadps4                                                               
Saving new configuration file /Users/marcinmitura/GitHub/shadPS4/build/user/config.toml
2024-08-31 14:01:56.192 shadps4[12000:3468403] +[IMKClient subclass]: chose IMKClient_Modern
2024-08-31 14:01:56.193 shadps4[12000:3468403] +[IMKInputSession subclass]: chose IMKInputSession_Modern
```

Reply:
Should be everything, I have no clue how to deal with translations that are compiled with app not created after first ran like user folder.